devito/finite_differences/finite_difference.py

```diff
@@ -351,66 +229,89 @@ def staggered_diff(expr, deriv_order, dim, fd_order, stagger=centered):
     fd_order : int
         Coefficient discretization order. Note: this impacts the width of
         the resulting stencil.
-    stagger : Side, optional
-        Shift of the finite-difference approximation.
+    stagger : Side of the staggered finite difference (left, right or centered)
```

Comment:
please, reuse exactly the same definition of `stagger` used in the other functions
also, it should be
```
stagger : tuple
    ...
```

Reply:
I also suggest to use finite-difference, not FD

Reply:
This is not a tuple here, this is the side of FD scheme `left, right` or `centered`

Reply:
sorry, `tuple` was just an example of a type. The typename must be short and self-explicative. 

Also, for consistency, we should use exactly what we used in other docstrings, e.g. from `second_derivative`:
```
    stagger : Side, optional
        Shift of the finite-difference approximation.
```